src/modules/client/mixins/helperMixin.js

```diff
@@ -143,7 +143,7 @@ export const helperMixin = {
         const userInfo = await Users.exists({ email: this.newHelper.local.email });
         const { user } = userInfo;
 
-        const sameOrNoCompany = !user.company || user.company === this.company._id;
+        const sameOrNoCompany = (!user.company && user._id) || user.company === this.company._id;
         if (userInfo.exists && (get(user, 'role.client') || !sameOrNoCompany)) {
           NotifyNegative('Utilisateur déjà existant');
         } else if (userInfo.exists) {
```

Comment:
est ce qu'on pourrait changer cette logique en passant la condition sur !userINfo.exists en premier ?
```
if (!userInfo.exists) this.firstStep = false;
        else if (get(user, 'role.client') || !sameOrNoCompany) NotifyNegative('Utilisateur déjà existant');
        else {
          const roles = await Roles.list({ name: HELPER });
          if (roles.length === 0) throw new Error('Role not found');

          const payload = { role: roles[0]._id, customer: this.customer._id };
          if (!user.company) payload.company = this.customer.company;

          await Users.updateById(user._id, payload);
          NotifyPositive('Aidant créé');

          this.getUserHelpers();
          this.openNewHelperModal = false;
        }
```

Reply:
et peux tu ajouter un await avant le `this.getUserHelpers();`

Reply:
done